<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QFrame" name="PatchExtractionFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Sans Serif&quot;;
text-align: center;
background-color: rgb(126, 126, 126);</string>
           </property>
           <property name="title">
            <string>Patch Extraction</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Dim</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="patch_type_comboBox">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>2D</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3D</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2.5D</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="styleSheet">
                   <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Save as </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="p_save_file_type_comboBox">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>.png</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>.npy</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Rows</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="patch_row">
                    <property name="value">
                     <number>32</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>column</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="patch_col">
                    <property name="value">
                     <number>32</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Depth</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="patch_depth"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Stride</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="stride_spinBox">
                    <property name="value">
                     <number>32</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="apply_to_folder_patchextratcion_checkBox">
              <property name="text">
               <string>To folder files</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>GT type:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="none_gt_radioButton">
                <property name="text">
                 <string>None</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="landmark_gt_radioButton">
                <property name="text">
                 <string>Landmark</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="mask_gt_radioButton">
                <property name="text">
                 <string>Image</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QPushButton" name="gt_mask_btn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Ground Truth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="load_landmarks_btn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Load Landmark</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="patch_generation_load_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Load Image</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Number of tumor patches:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="num_tumor_patches">
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Number of non tumor patches:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="num_ntumor_patches">
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="select_patch_saving_folder_btn">
                <property name="text">
                 <string>Destination folder</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="extract_patches_btn">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 68, 68);
background-color: rgb(166, 166, 166);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider">
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(102, 102, 102);</string>
           </property>
           <property name="title">
            <string>Image Augmentation</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="styleSheet">
                     <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Angle</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="rotation_angle_spinBox"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="flip_vertical_checkBox">
                  <property name="text">
                   <string> Flip vertical</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="flip_horizontal_checkBox">
                  <property name="text">
                   <string>Flip horizontal</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="load_aug_landmark_checkBox">
                  <property name="text">
                   <string>Landmark</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="apply_to_folder_aug_checkBox">
                  <property name="text">
                   <string>To folder files</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QPushButton" name="aug_load_landmarks_btn">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Load Landmark</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="aug_load_img_btn">
                    <property name="text">
                     <string>Load Image</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QPushButton" name="aug_dst_folder_btn">
                    <property name="text">
                     <string>Destination folder</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="aug_apply_btn">
                    <property name="text">
                     <string>Apply</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Sans Serif&quot;;
background-color: rgb(191, 200, 188);</string>
           </property>
           <property name="title">
            <string>Other Preprocessing</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Histogram Equalization</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QRadioButton" name="histogram_equa_radio_btn">
                  <property name="text">
                   <string>Non adaptive</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="adap_histogram_equa_radio_btn">
                  <property name="text">
                   <string>Adaptive</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QCheckBox" name="hist_match_checkBox">
                <property name="text">
                 <string>Histogram Matching</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="load_ref_img_btn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Load Reference Image</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QCheckBox" name="apply_to_folder_otherprepro_checkBox">
                <property name="text">
                 <string>To folder files</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="normalization_checkBox">
                <property name="text">
                 <string>Normalization</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="otherpreprocessing_load_img_btn">
                <property name="text">
                 <string>Load Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="otherpreprocessing_dst_folder_btn">
                <property name="text">
                 <string>Destination folder</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="otherpreprocessing_apply_btn">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="statuslabel">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <action name="actionPatch_generate">
   <property name="text">
    <string>patch generate</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionAugment">
   <property name="text">
    <string>augment</string>
   </property>
  </action>
  <action name="actionOpen_image">
   <property name="text">
    <string>load image</string>
   </property>
   <property name="iconText">
    <string>open image</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLoad_landmark">
   <property name="text">
    <string>load landmark</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAbout_Patch_Generation">
   <property name="text">
    <string>About Patch Generation</string>
   </property>
  </action>
  <action name="actionAbout_Image_Augmentation">
   <property name="text">
    <string>About Image Augmentation</string>
   </property>
  </action>
  <action name="actionQuitapplication">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/rsc.qrc"/>
 </resources>
 <connections/>
</ui>
